hadoop-ozone/dist/src/main/compose/ozonesecure-mr/docker-compose.yaml

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-version: "3"
+version: "3.5"
```

Comment:
Do we track the docker-compose versions for this change?

Reply:
Docker Compose file [version 3.5](https://docs.docker.com/compose/compose-file/compose-versioning/#version-35) is the first to allow `name` for networks.